core/electron/src/frontend/ElectronApp.ts

```diff
@@ -5,32 +5,40 @@
 import { ProcessDetector, PromiseReturnType } from "@itwin/core-bentley";
 import { IpcListener, IpcSocketFrontend } from "@itwin/core-common";
 import { IpcApp, NativeApp, NativeAppOpts } from "@itwin/core-frontend";
-import { ITwinElectronApi } from "../backend/ElectronPreload";
+import type { IpcRenderer } from "electron";
 import { DialogModuleMethod } from "../common/ElectronIpcInterface";
 import { ElectronRpcManager } from "../common/ElectronRpcManager";
+import type { ITwinElectronApi } from "../common/ITwinElectronApi";
+
+declare global {
+  interface Window {
+    itwinjs: ITwinElectronApi;
+  }
+}
 
 /**
  * Frontend Ipc support for Electron apps.
  */
 class ElectronIpc implements IpcSocketFrontend {
-  private _api: ITwinElectronApi;
+  private _api: ITwinElectronApi | IpcRenderer;
   public addListener(channelName: string, listener: IpcListener) {
-    this._api.addListener(channelName, listener);
-    return () => this._api.removeListener(channelName, listener);
+    this._api?.addListener(channelName, listener);
```

Comment:
I don't understand why the `?` was introduced - can `_api: ITwinElectronApi | IpcRenderer` ever be null or undefined?

Reply:
sorry this was left over from debugging, removed